rdrf/rdrf/admin.py

```diff
@@ -470,3 +471,28 @@ class CDEFileAdmin(admin.ModelAdmin):
 admin.site.register(Notification, NotificationAdmin)
 admin.site.register(DemographicFields, DemographicFieldsAdmin)
 admin.site.register(ConsentRule, ConsentRuleAdmin)
+
+
```

Comment:
PROMS_ADMIN_INTERFACE = [ (Survey, SurveyAdmin), ....]
DESIGN_MODE_ADMIN_INTERFACE = [ (RegistryCommonDataElement, ..),...]
NORMAL_ADMIN_INTERFACE = [(...), (...),]

if settings.PROMS:
    ADMIN_INTERFACE = PROMS_ADMIN_INTERFACE
else:
   ADMIN_INTERFACE = NORMAL ...
if settings.DESIGN_MODE:
   ADMIN_INTERFACE = ADMIN_INTERFACE + DESIGN_MODE_INTERFACE

for model_class, model_admin in ADMIN_INTERFACE:
       register them